pkgs/development/python-modules/paddlepaddle/default.nix

```diff
@@ -0,0 +1,71 @@
+{ stdenv
+, lib
+, buildPythonPackage
+, fetchPypi
+, python
+, pythonOlder
+, pythonAtLeast
+, autoPatchelfHook
+, openssl_1_1
+, zlib
+, setuptools
+# runtime dependencies
+, httpx
+, numpy
+, protobuf
+, pillow
+, decorator
+, astor
+, paddle-bfloat
+, opt-einsum
+}:
+
+let
+  pname = "paddlepaddle";
+  version = "2.5.0";
+  format = "wheel";
+  pyShortVersion = "cp${builtins.replaceStrings ["."] [""] python.pythonVersion}";
+  hashAndPlatform = import ./binary-hashes.nix { pythonVersion = pyShortVersion; system = stdenv.system; };
+  src = fetchPypi ({
+    inherit pname version format;
+    dist = pyShortVersion;
+    python = pyShortVersion;
+    abi = pyShortVersion;
+  } // hashAndPlatform);
+in
+buildPythonPackage {
+  inherit pname version format src;
+
+  disabled = pythonOlder "3.9" || pythonAtLeast "3.11";
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+  ];
+
+  buildInputs = [
+    openssl_1_1
```

Comment:
This was suggested as a dirty fix here: https://github.com/NixOS/nixpkgs/pull/243583#discussion_r1267562718.

A potentially better solution was suggested here: https://github.com/NixOS/nixpkgs/pull/243583#issuecomment-1641450848. I tried that approach for some time, before finally giving up after being unable to follow [the linked build instructions](https://www.paddlepaddle.org.cn/documentation/docs/en/install/compile/linux-compile_en.html#compile-on-linux-from-source-code).

Reply:
maybe let's just add a comment with TODO and a link to the linux build instruction.